website/server/libs/cron.js

```diff
@@ -329,11 +329,11 @@ export function cron (options = {}) {
     if (resetWeekly === true && resetMonthly === true) {
       break;
     }
-    let thatDay = moment(now).subtract({days: i}).toDate();
-    if (thatDay.getDay() === 1) {
+    let thatDay = moment(now).utcOffset(user.preferences.timezoneOffset).subtract({days: i});
```

Comment:
I'm pretty sure this needs to be zone because we haven't been upgrading to utcOffset.